wsmaster/che-core-api-auth-bitbucket/src/main/java/org/eclipse/che/security/oauth/BitbucketOAuthAuthenticator.java

```diff
@@ -108,5 +110,8 @@ private void testRequest(String requestUrl, String accessToken)
         urlConnection.disconnect();
       }
     }
+    if (isNullOrEmpty(result)) {
```

Comment:
It is possible to happen?

Reply:
Yes, the `whoami` API request with an invalid token returns an empty response.